### PR TITLE
Change url from dca to tarc

### PR DESCRIPTION
**Pull request by rawOrlando**
Splitting off this part of #57.
Will take care of https://tuftswork.atlassian.net/browse/TDLR-2454

### CODE REVIEW

**Comment by rawOrlando** [comment]
Are we also changing `sites.tufts.edu/dca/`

**Review by mkorcy — CHANGES_REQUESTED** [review]
I requested a few minor changes, talked to Adrienne about the appropriate mapping for the http://dca.tufts.edu/ua/access/rights-pd.html URL in our fixtures, and the one I listed in comments is correct. And also, see my last note on the election fixture.

File: lib/tufts/populate_fixtures.rb [hunk 1]
```diff
@@ -310,7 +310,7 @@ def create_object(pid, metadata)
         date_available: "2011-06-17",
         resource_type: "http://purl.org/dc/dcmitype/Text",
         format_label: "text/xml",
-        rights_statement: "http://dca.tufts.edu/ua/access/rights-pd.html",
+        rights_statement: "http://tarc.tufts.edu/ua/access/rights-pd.html",
```
[the remark this answers is not in the document]
Comment (rawOrlando) [reply lib/tufts/populate_fixtures.rb:313]: Great! the old one also 404. But fixing that problem too!

File: spec/fixtures/tufts_il.president.1824.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<aas:election_record xmlns:aas="http://dca.tufts.edu/aas" label="Illinois 1824 President of the United States" type="Legislative" iteration="First Ballot" date="1824-11" election_id="il.president.1824" handle="10427/56366" schemaLocation="http://dca.tufts.edu/aas http://dca.tufts.edu/schema/aas/electionRecord.xsd">
+<aas:election_record xmlns:aas="http://dca.tufts.edu/aas" label="Illinois 1824 President of the United States" type="Legislative" iteration="First Ballot" date="1824-11" election_id="il.president.1824" handle="10427/56366" schemaLocation="http://dca.tufts.edu/aas http://tarc.tufts.edu/schema/aas/electionRecord.xsd">
```
[the remark this answers is not in the document]
Comment (rawOrlando) [reply spec/fixtures/tufts_il.president.1824.xml:2]: Ok, I can get rid of this change. But do you want to keep the changes to the other fixtures? MS226, and MS999?

[the remark this answers is not in the document]
Comment (mkorcy) [reply spec/fixtures/tufts_il.president.1824.xml:2]: Yeah, EAD fixtures I'm not worried about, they're going to go away soon anyway, but the NNV/AAS stuff is sticking around.

**Review by mkorcy — APPROVED** [review]
Good to go

**Pull request merged**